Pages/Index.cshtml.cs

```diff
@@ -13,40 +13,69 @@ namespace NewYork_CIty_School_Data_With_Crime_Rate_History.Pages
 {
     public class IndexModel : PageModel
     {
+        public string GetData(string endpoint)
+        {
+            string downloadedData = "";
+            using (WebClient webClient = new WebClient())
+            {
+                downloadedData = webClient.DownloadString(endpoint);
+            }
+            return downloadedData;
+        }
         public JsonResult OnGet()
         {
 
             List<MergedData> mergedData = new List<MergedData>();
 
             using (WebClient webClient = new WebClient())
             {
-                string schoolJsonString = webClient.DownloadString("https://data.cityofnewyork.us/resource/23z9-6uk9.json");
+                string schoolJsonString = GetData("https://data.cityofnewyork.us/resource/23z9-6uk9.json");
+                //               string schoolJsonString = webClient.DownloadString("https://data.cityofnewyork.us/resource/23z9-6uk9.json");
```

Comment:
Remove this entirely.  It's considered bad practice to commit commented-out code to version control, because you can always go back in history to see what used to be there.